submissions/chris-voitova/friends-app/script.js

```diff
@@ -0,0 +1,206 @@
+const URL = "https://randomuser.me/api/?results=16";
+const USERS_CONTAINER = document.querySelector(".users-container");
+
+const INIT_GENDER = "all";
+
+let selectedGender = INIT_GENDER;
+
+const showMessage = (message) => `<p class="message">${message}</p>`;
+
+const createUserTemplate = (photo, name, email, gender, age) => {
+  return `
+   <div class="user">
+      <img
+         class="user__photo"
+         src=${photo}
+         alt="user-photo"
+      />
+      <div class="user__info">
+         <h3 class="user__name">${name}</h3>
+         <a class="user__email" href="mailto:${email}">
+           ${email}
+         </a>
+         <span class="user__gender">gender: ${gender}</span>
+         <span class="user__age">age: ${age}</span>
+      </div>
+   </div>`;
+};
+
+async function getUsers() {
+  try {
+    const response = await fetch(URL);
```

Comment:
Please, handle fetch status :) https://www.tjvantoll.com/2015/09/13/fetch-and-errors/

Reply:
done